config/index.js

```diff
@@ -310,6 +310,18 @@ var conf = convict({
       }
     }
   },
+  mailerServer: {
+    host: {
+      doc: 'Host for mailer_server.js',
+      format: 'ipaddress',
+      default: '127.0.0.1'
```

Comment:
Doesn't matter now, but exposing a MAILER_LISTEN_HOST (or perhaps, MAILER_LISTEN_IP_ADDRESS) would be good.